src/shared/components/data-table/components/main/components/table-header/TableHeader.scss

```diff
@@ -0,0 +1,69 @@
+@import 'src/styles/common';
+
+.header {
+  height: 30px;
```

Comment:
Dangerous. What if a column header is taller?

Reply:
Leftovers from some experiments I did previously. Removed it